lib/nexpose/common.rb

```diff
@@ -107,7 +142,7 @@ class Schedule < APIObject
     # The date after which the schedule is disabled, in ISO 8601 format.
     attr_accessor :not_valid_after
 
-    #TODO Remove this unused attribute
+    # TODO: Remove this unused attribute
```

Comment:
The XML API supports this optional field. The backend code reads it but it doesn't do anything special.